tests/testImageMapReduce.py

```diff
@@ -167,71 +166,113 @@ def testExampleTaskWithOverlaps(self):
         newArr = newMI.getImage().getArray()
         mi = self.exposure.getMaskedImage()
 
-        self.assertClose(mi.getImage().getArray().mean(), newArr.mean() - 5.)
-        self.assertClose(newArr.mean(), 5.)
-        self.assertClose(newArr.min(), 5.)
-        self.assertClose(newArr.max(), 5.)
+        self.assertFloatsAlmostEqual(mi.getImage().getArray(), newArr - 5.)
 
     def testExampleTaskMean(self):
         """Test sample grid task that returns the mean of the subimages and uses
         'none' `reduceOperation`.
         """
-        config = TestImageMapReduceConfig2()
+        config = GetMeanImageMapReduceConfig()
         config.gridStepX = config.gridStepY = 8
         config.reducerSubtask.reduceOperation = 'none'
         task = ImageMapReduceTask(config)
-        subMeans = task.run(self.exposure)
+        testExposure = self.exposure.clone()
+        testExposure.getMaskedImage()[:, :] = 1.234
+        subMeans = task.run(testExposure)
 
         self.assertEqual(len(subMeans), len(task.boxes0))
-        self.assertClose(subMeans[0], 0.)
+        self.assertFloatsAlmostEqual(np.array(subMeans), 1.234, rtol=1e-6)
 
     def testGridCentroids(self):
         """Test sample grid task which is provided a set of `gridCentroids` and
         returns the mean of the subimages surrounding those centroids using 'none'
         for `reduceOperation`.
         """
-        config = TestImageMapReduceConfig2()
+        config = GetMeanImageMapReduceConfig()
         config.gridStepX = config.gridStepY = 8
         config.reducerSubtask.reduceOperation = 'none'
         centroidsX = [i for i in np.linspace(0, 128, 50)]
         centroidsY = centroidsX
         config.gridCentroidsX = centroidsX
         config.gridCentroidsY = centroidsY
         task = ImageMapReduceTask(config)
-        subMeans = task.run(self.exposure)
+        testExposure = self.exposure.clone()
+        testExposure.getMaskedImage()[:, :] = 1.234
+        subMeans = task.run(testExposure)
 
         self.assertEqual(len(subMeans), len(centroidsX))
-        self.assertClose(subMeans[0], 0.)
+        self.assertFloatsAlmostEqual(np.array(subMeans), 1.234, rtol=1e-6)
 
     def testGridValidity(self):
-        """!Test sample grids with various spacings and sizes and other options.
+        """Test sample grids with various spacings and sizes and other options.
 
         Try to see if we can break it.
```

Comment:
I'm not sure this line of the docstring is actually helpful? Maybe if you defined what this could break, and how?